css/syntaxes.md

```diff
@@ -3,39 +3,45 @@
 [data](https://github.com/mdn/data/blob/master/css/syntaxes.json) |
 [schema](https://github.com/mdn/data/blob/master/css/syntaxes.schema.json)
 
-[CSS basic data types](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Types) define the kinds of values (keywords and units) accepted by CSS properties and functions.
+[CSS value definition syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/Value_definition_syntax) is used for the formal syntax of CSS properties. The syntaxes.json file defines many of these CSS syntaxes.
+
+For example, the `background-attachment` property has the following syntax where `<attachment>` is referring to a syntax that is defined in this file.
```

Comment:
Instead of "in this file", I'd say "in syntaxes.json", just to make things crystal clear.